docs/examples/multimodal_classification/Multimodal_Classification_with_Ludwig_Python_API.ipynb

```diff
@@ -0,0 +1,388 @@
+        "|-------------|------|-----------------------------------------------------|\n",
+        "| default_profile | binary | Boolean indicating whether the account has a default profile |\n",
+        "| default_profile_image | binary | Boolean indicating whether the account has a default profile image |\n",
+        "| description | text |  User account description                           |\n",
+        "| favorites_count | number | Total number of favourited tweets             |\n",
+        "| followers_count | number | Total number of followers                     |\n",
+        "| friends_count | number | Total number of friends                         |\n",
+        "| geo_enabled | binary | Boolean indicating whether the account has the geographic location enabled  |\n",
+        "| lang | category | Language of the account                                |\n",
+        "| location | category | Location of the account                            |\n",
+        "| profile_background_image_url | image | Profile background image url      |\n",
+        "| profile_image_url | image | Profile image url                            |\n",
+        "| statuses_count | number | Total number of tweets                         |\n",
+        "| verified | binary | Boolean indicating whether the account has been verified |\n",
+        "| average_tweets_per_day | number | Average tweets posted per day          |\n",
+        "| account_age_days | number | Account age measured in days                 |\n",
+        "| account_type   | category | Account type, one of {bot, human}            |\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "jclY9wwNCIJM"
+      },
+      "outputs": [],
+      "source": [
+        "# Downloads the dataset to the current working directory\n",
+        "!kaggle datasets download danieltreiman/twitter-human-bots-dataset\n",
+        "\n",
+        "# Unzips the downloaded dataset, creates twitter_human_bots_dataset.csv\n",
+        "!unzip -q -o twitter-human-bots-dataset.zip"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "v2gsXFVTClIN"
+      },
+      "outputs": [],
+      "source": [
+        "import pandas as pd\n",
+        "\n",
+        "# Loads the dataset\n",
+        "dataset_df = pd.read_csv('twitter_human_bots_dataset.csv')\n",
+        "\n",
+        "# Previews a few rows of the dataset\n",
+        "dataset_df.head(5)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "FczUptADgreZ"
+      },
+      "outputs": [],
+      "source": [
+        "# Prerequisite: Installs the latest version of Ludwig in the Colab environment\n",
+        "!python -m pip install git+https://github.com/ludwig-ai/ludwig.git --quiet"
```

Comment:
When I tried running the notebook, I encountered this error.  Somehow it seemed to pickup the TF install not PyTorch?
```
Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
     |████████████████████████████████| 6.0 MB 5.5 MB/s 
     |████████████████████████████████| 408 kB 49.5 MB/s 
     |████████████████████████████████| 47 kB 4.3 MB/s 
     |████████████████████████████████| 4.0 MB 36.0 MB/s 
     |████████████████████████████████| 136 kB 47.6 MB/s 
     |████████████████████████████████| 181 kB 48.4 MB/s 
     |████████████████████████████████| 10.1 MB 35.0 MB/s 
     |████████████████████████████████| 653 kB 42.0 MB/s 
     |████████████████████████████████| 457 kB 42.9 MB/s 
     |████████████████████████████████| 42 kB 1.1 MB/s 
     |████████████████████████████████| 6.6 MB 40.0 MB/s 
     |████████████████████████████████| 596 kB 37.2 MB/s 
     |████████████████████████████████| 77 kB 5.3 MB/s 
     |████████████████████████████████| 895 kB 38.3 MB/s 
     |████████████████████████████████| 1.1 MB 30.0 MB/s 
     |████████████████████████████████| 144 kB 51.9 MB/s 
     |████████████████████████████████| 271 kB 53.6 MB/s 
     |████████████████████████████████| 94 kB 186 kB/s 
  Building wheel for ludwig (PEP 517) ... done
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
tensorflow 2.8.0 requires tf-estimator-nightly==2.8.0.dev2021122109, which is not installed.
```

Reply:
I get that pip error too, but I don't think it has anything to do with Ludwig - I think that is coming from the Colab environment

Reply:
Strange we get it on this notebook but not the others.